components/clp-package-utils/clp_package_utils/scripts/native/decompress.py

```diff
@@ -25,15 +25,24 @@
 logger.addHandler(logging_console_handler)
 
 
-def decompress_paths(
+def handle_decompression(
+    parsed_args,
     clp_home: pathlib.Path,
-    paths,
-    list_path: pathlib.Path,
     clp_config: CLPConfig,
-    archives_dir: pathlib.Path,
-    logs_dir: pathlib.Path,
-    extraction_dir: pathlib.Path,
 ):
```

Comment:
```suggestion
):
    """
    Handles the decompression command.
    :param parsed_args:
    :param clp_home:
    :param clp_config:
    :return: 0 on success, -1 otherwise.
    """
```